thundra/plugins/log/thundra_log_handler.py

```diff
@@ -7,17 +10,24 @@ class ThundraLogHandler(logging.Handler):
     def __init__(self):
         logging.Handler.__init__(self)
         ThundraLogHandler.logs = []
+        self.tracer = ThundraTracer.getInstance()
+        self.scope = None
 
     def emit(self, record):
         formatted_message = self.format(record)
+        active_span = self.tracer.get_active_span()
```

Comment:
If trace plugin is not active, `active_span` is not available. So there should be check for this case for getting informations from this variable below.